app/components/dashboard/dashboard-controller.ts

```diff
@@ -77,7 +87,7 @@ export class DashboardController {
          this.totalCapacity = {
                 freeGB: 0, usedGB: 0, totalGB: 0,
                 freeFormatted: '0 B', usedFormatted: '0 B', totalFormatted: '0 B'
-            };
+         };
```

Comment:
ya , we can remove initial value for totalCapacity . But it's being used . and using this only i am getting usedSize, freeSize , we can't remove this variable completely.